client/core/core.go

```diff
@@ -1512,6 +1512,12 @@ func (c *Core) Run(ctx context.Context) {
 	// Store the context as a field, since we will need to spawn new DEX threads
 	// when new accounts are registered.
 	c.ctx = ctx
+	defer func() {
+		if pv := recover(); pv != nil {
+			c.log.Critical(pv)
+			close(c.ready) // unblock <-Ready()
```

Comment:
mm, maybe I will just add and error return to `initialize` since it's possible the channel was already closed if it panicked *after* initialize rather than in it... meh